integration-tests/originate-known-contracts.ts

```diff
@@ -135,7 +135,7 @@ CONFIGS().forEach(({ lib, setup, protocol }) => {
     });
 
     // originate tx rollup
-    if (protocol === Protocols.ProtoALpha) {
+    if (protocol === Protocols.ProtoALpha || protocol === Protocols.PtMumbaii) {
```

Comment:
I think we might not need this anymore since TORU is now disabled in Mumbai(?)

Reply:
If we run this file now it will try to originate tx rollup contract in mumbainet is the reason I added that condition, to just leave it as empty string like ProtoAlpha is now. It will be even better we remove tx rollup origination script from that file and remove the references in config of integration-tests and all integration tests.

Reply:
yeah exactly, I think we should perhaps remove it if the contract isn't even being used. Might need to double check for that though

Reply:
I didn't remove the txrollup script there cuz I wasn't sure how much it will affect our integration-test config and related integration tests. Is this something we'd like to create a ticket for or I just ship it with this pr?